components/concourse-github-resource/assets/in

```diff
@@ -146,6 +135,6 @@ if [[ "${approval_count}" -lt "${required_approval_count}" ]]; then
   exit 1
 fi
 
-echo "[SUCCESS] You have ${approval_count} GitHub approval(s) and needed ${required_approval_count} or more."
+echo "[SUCCESS] You have ${valid_approval_count} GitHub approval(s) and needed ${required_approval_count} or more."
```

Comment:
I guess this technically isn't true because we don't exclude the person who raised the PR but I guess it's good enough for now.

Reply:
Good catch. It's not possible to Approve your own PR.